packages/tooling/fast-tooling/webpack.dev.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-undef */
+const merge = require("webpack-merge");
+const baseConfig = require("./webpack.common");
+
+module.exports = merge(baseConfig, {
+    devServer: {
+        contentBase: "./src/public",
+        open: true,
+        port: 7776,
```

Comment:
Are we using the same port value as other local web servers?

Reply:
Potentially, but in theory you wouldn't be running these at the same time.